packages/pipeline-editor/src/SubmitNotebook.ts

```diff
@@ -1,313 +0,0 @@
- *  - Attach button to notebook toolbar and launch a dialog requesting
- *  information about the remote location to where submit the notebook
- *  for execution
- */
-export class SubmitNotebookButtonExtension
-  implements DocumentRegistry.IWidgetExtension<NotebookPanel, INotebookModel> {
-  private panel: NotebookPanel;
-
-  constructor(app: JupyterFrontEnd) {
-    this.app = app;
-  }
-
-  readonly app: JupyterFrontEnd;
-
-  showWidget = async (): Promise<void> => {
-    const envVars: string[] = NotebookParser.getEnvVars(
-      this.panel.content.model.toString()
-    );
-
-    const runtimes = await PipelineService.getRuntimes();
-    const runtimeImages = await PipelineService.getRuntimeImages();
-
-    showDialog({
-      title: 'Submit notebook',
-      body: new SubmitNotebook(envVars, runtimes, runtimeImages),
-      buttons: [Dialog.cancelButton(), Dialog.okButton()]
-    }).then(result => {
-      if (result.value == null) {
-        // When Cancel is clicked on the dialog, just return
-        return;
-      }
-
-      // prepare notebook submission details
-      const notebookOptions: ISubmitNotebookOptions = result.value as ISubmitNotebookOptions;
-      const pipeline = Utils.generateNotebookPipeline(
-        this.panel.context.path,
-        notebookOptions
-      );
-
-      PipelineService.submitPipeline(
-        pipeline,
-        PipelineService.getDisplayName(result.value.runtime_config, runtimes)
-      );
-    });
-  };
-
-  createNew(
-    panel: NotebookPanel,
-    context: DocumentRegistry.IContext<INotebookModel>
-  ): IDisposable {
-    this.panel = panel;
-
-    // Create the toolbar button
-    const submitNotebookButton = new ToolbarButton({
-      label: 'Submit Notebook ...',
-      onClick: this.showWidget,
-      tooltip: 'Submit Notebook ...'
-    });
-
-    // Add the toolbar button to the notebook
-    panel.toolbar.insertItem(10, 'submitNotebook', submitNotebookButton);
-
-    // The ToolbarButton class implements `IDisposable`, so the
-    // button *is* the extension for the purposes of this method.
-    return submitNotebookButton;
-  }
-}
-
-/**
- * Submit notebook dialog extension
- * - Request information about the remote location to where submit the
- * notebook for execution
- */
-export class SubmitNotebook extends Widget
-  implements Dialog.IBodyWidget<ISubmitNotebookConfiguration> {
-  _envVars: string[];
-  _runtimes: any;
-  _runtimeImages: IDictionary<string>;
-
-  constructor(
-    envVars: string[],
-    runtimes: any,
-    runtimeImages: IDictionary<string>
-  ) {
-    super();
-
-    this._envVars = envVars;
-    this._runtimes = runtimes;
-    this._runtimeImages = runtimeImages;
-
-    this.node.appendChild(this.renderHtml());
-    (this.node.getElementsByClassName(
-      'elyra-form-runtime-config'
-    )[0] as HTMLSelectElement).value = '';
-    (this.node.getElementsByClassName(
-      'elyra-form-framework'
-    )[0] as HTMLSelectElement).value = '';
```

Comment:
@lresende this PR removes the code that explicitly sets the drop downs default values. IIUC the drop downs will use the first value as default after this change (I will double check when I test this locally). IS this change in behavior ok? We've discussed it in the past but I can't remember the details